blackbox/docs/best_practice/insert_performance.txt

```diff
@@ -37,4 +37,9 @@ If you use the :ref:`sql_http_endpoint`, you can insert multiple records at
 once by making use of :ref:`bulk_operations`. Our benchmarking_ indicates that
 you can expect to see at least a twofold increase in ingestion performance.
 
+Bulk inserts still generate internal network traffic, and CrateDB still waits
```

Comment:
Maybe mention also that a key thing why bulk inserts are faster is that the query needs to be only parsed and processed once by the backend. With single inserts this progress needs to be done every time.